### ISSUE TITLE
Fatal error on installation with Sample Data

### ISSUE THREAD

**Issue by vkublytskyi** [magento/inventory#1561]
### Preconditions

1. A fresh copy of Magento 2 from `magento-engcom/msi` of `2.3-develop` branch is installed

### Steps to reproduce

1. Clone [Magento Open Source Sample Data](https://github.com/magento/magento2-sample-data)
1. Link Sample Data (see [DevDocs](https://devdocs.magento.com/guides/v2.3/install-gde/install/sample-data-after-clone.html))
1. Run magento upgrade command `php bin/magento setup:upgrade`

### Expected result

1. Magento upgraded
1. Sample data installed

### Actual result

1. Fatal exception occurred:

```
Module 'Magento_SalesSampleData':
Installing data... PHP Fatal error:  Uncaught Error: Call to a member function getQuantity() on null in /magento2/magento2ce/app/code/Magento/InventorySourceDeductionApi/Model/SourceDeductionService.php:85
```

**Comment by TadeuRodrigues** [comment]
wich your php version? I will test.

**Comment by vkublytskyi** [comment]
@TadeuRodrigues PHP 7.2

**Comment by TadeuRodrigues** [comment]
ok, I'm testing.

**Comment by TadeuRodrigues** [comment]
Did u do that? https://devdocs.magento.com/guides/v2.3/install-gde/install/sample-data-after-clone.html

**Comment by IvanPletnyov** [comment]
Hi. It's dublicate [issue 1438](https://github.com/magento-engcom/msi/issues/1438)

**Comment by vkublytskyi** [comment]
Closed as is duplication of #1438
